cairo/ethereum/cancun/fork.cairo

```diff
@@ -292,7 +296,7 @@ func validate_header{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr:
         );
         assert number_is_valid = 1;
 
-        let extra_data_is_valid = is_zero(32 - header.value.extra_data.value.len);
+        let extra_data_is_valid = is_le_felt(header.value.extra_data.value.len, 32);
```

Comment:
are we expecting headers of more than 2**128 len?

Reply:
No it should be max 32bytes. But there is no validation prior to this. So I am not sure we can use is_le ?

Reply:
is_le fails if the argument is > range check bound so we should be using it imo